<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KitapEkleForm</class>
 <widget class="QWidget" name="KitapEkleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="kitapAdiLabel">
      <property name="text">
       <string>Kitap Adı</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="kitapAdiLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="kitapYazariLabel">
      <property name="text">
       <string>Kitap Yazarı</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="kitapYazariLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="okunmaDurumuLabel">
      <property name="text">
       <string>Okunma Durumu</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="okunmaDurumuCheckBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="begeniDurumuLabel">
      <property name="text">
       <string>Beğeni Durumu</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="begeniDurumuLineEdit">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="kayitDurumuLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kayıt Durumu</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="kayitButonu">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kitap Kaydet</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
